src/hooks/useLoginForm.ts

```diff
@@ -0,0 +1,45 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import * as z from "zod";
+
+// Data for checking validate login form
+const dataDummy = {
+  id: 1,
+  email: "dummy@gmail.com",
+  password: "Admin123",
+};
+
+const formSchema = z.object({
+  email: z.string().email({ message: "Invalid email address" }),
+  password: z.string().min(1, { message: "Invalid password" }),
+});
+
+export type LoginFormValues = z.infer<typeof formSchema>;
+
+const useLoginForm = (): any => {
```

Comment:
Do not use `any` here as this will remove the intellisense. Make an interface as the return type
```ts
interface ILoginFormProps {
  form: UseFormReturn<LoginFormValues>;
  onSubmit: (values: LoginFormValues) => void;
}
```